index.bs

```diff
@@ -203,18 +203,26 @@ for tracking sites is cleared after 24 hours, unless the user has interacted
 with the site in the first-party context in the last 45 days.
 
 <h4 id="mitigations-brave">Brave</h4>
-Brave uses three list-based approaches to combat [=navigational tracking=].
+Brave uses four list-based approaches to combat [=navigational tracking=].
 
 First, Brave strips query parameters commonly used for [=navigational tracking=]
 from URLs on navigation. This list is maintained by Brave.
 
-Second, in the non-default, "aggressive blocking" configuration, Brave uses
+Second, by default, when i) the user is about to visit a list-identified
+bounce-tracking site, and ii) the current profile does not contain any cookies
+or DOM state for that site, Brave will create a new, "ephemeral", empty storage
```

Comment:
How would a user get cookies or other storage for a site on this list? It seems like with this mitigation in place, such storage would always get deleted after the user's visit, and so this "does not contain" condition would always be true. Is this meant to capture when the user has a tab still open on this site, and they open a second one, or am I missing a possibility?

Reply:
Ah i see your point, i should clarify the text.  The the lists we use classify URLs as bounce-tracking (or not), so its possible to have `||tracker.example/bouncer` labeled as a bounce tracking page, but the rest of `tracker.example` unlabeled.  So, if a user visits, say https://tracker.example/index.html at some point, and storage is set, and then later is bounced across https://tracker.example/bouncer?dest=<url>, the mitigation would not be applied (in this current version of the feature)